discord-seed-call.js

```diff
@@ -4,7 +4,7 @@ import DiscordBasePlugin from './discord-base-plugin.js';
 
 export default class DiscordSeedCall extends DiscordBasePlugin {
   static get description() {
-    return 'The <code>DiscordSeedCall</code> plugin will send a message in a Discord channel at specified time of the day.';
+    return `The <code>DiscordSeedCall</code> plugin will send a message in a Discord channel at specified time of the day.`;
```

Comment:
I would very much prefer to not have template strings everywhere. That construct is specifically for string interpolation.